code/game/objects/machinery/vending/new_marine_vendors.dm

```diff
@@ -1150,3 +1261,6 @@ GLOBAL_LIST_INIT(available_specialist_sets, list("Scout Set", "Sniper Set", "Dem
 
 #undef MARINE_CAN_BUY_ALL
 #undef MARINE_TOTAL_BUY_POINTS
+
+#undef SQUAD_LOCK
+#undef JOB_LOCk
```

Comment:
```suggestion
```

Reply:
Undefs are good if the define is only use in one file, unless there's some other reason they shouldn't be used

Reply:
It was causing a compile error

Reply:
That's because the `k` is lower case. Fix it instead of removing it.